com.unity.multiplayer.transport.utp/Tests/Editor/BasicUTPTest.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using Unity.Networking.Transport;
+
```

Comment:
Would it be practical to add a test where we standup 2 UTPTransport instances and send to each other?

Reply:
Yes, we will do that :P as mentioned before had to add testing to get validation passing.